schemas/coordinator.json

```diff
@@ -0,0 +1,211 @@
+{
+    "openrpc": "1.2.6",
+    "info": {
+        "title": "Coordinator",
+        "description": "Coordinates messages between LECO Components.",
+        "version": "0.1.0"
+    },
+    "methods": [
+        {
+            "name": "sign_in",
+            "params": [],
+            "description": "Sign in to this Coordinator.",
+            "result": {
+                "name": "result",
+                "schema": {
+                    "type": "null"
+                },
+                "required": true
+            },
+            "errors": [
+                {
+                    "code": -32091,
+                    "message": "The name is already taken.",
+                    "data": "Already taken name."
+                }
+            ]
```

Comment:
Connect it to the error definition, lest this definition and the error list diverge.